bin/gen-pgbouncer-conf.sh

```diff
@@ -56,6 +56,7 @@ log_connections = ${PGBOUNCER_LOG_CONNECTIONS:-1}
 log_disconnections = ${PGBOUNCER_LOG_DISCONNECTIONS:-1}
 log_pooler_errors = ${PGBOUNCER_LOG_POOLER_ERRORS:-1}
 stats_period = ${PGBOUNCER_STATS_PERIOD:-60}
+stats_users = ${PGBOUNCER_STATS_USER}
```

Comment:
thanks for noticing, I will change the code to check for  `PGBOUNCER_MONITORING_ENABLED` env variable before setting this values.